### PR TITLE
[test] Update automatic_arrays_acc

### PR DESCRIPTION
**Pull request by ajocksch**

### CODE REVIEW

File: cscs-checks/mch/automatic_arrays_acc.py [hunk 1]
```diff
@@ -46,6 +49,13 @@ def __init__(self):
 
     @run_before('compile')
     def setflags(self):
+        if self.current_system.name in ['daint', 'dom']:
+            if (not self.current_environ.name.startswith('PrgEnv-nvidia')):
```
Comment (teojgo) [inline cscs-checks/mch/automatic_arrays_acc.py:53]: The outer parentheses are not needed.

File: cscs-checks/mch/automatic_arrays_acc.py [hunk 2]
```diff
@@ -12,10 +12,9 @@
 class AutomaticArraysCheck(rfm.RegressionTest):
     def __init__(self):
```
Comment (teojgo) [inline cscs-checks/mch/automatic_arrays_acc.py:13]: Can you also adapt so that this uses class level attrributes instead of the older `__init__` method?

**Review by teojgo — CHANGES_REQUESTED** [review]

**Review by jjotero — APPROVED** [review]
lgtm

**Comment by jjotero** [comment]
@jenkins-cscs retry tsa

**Comment by jjotero** [comment]
@jenkins-cscs retry daint

**Pull request merged**